### ISSUE TITLE
Setting tags with `send_raw_email/4`

### ISSUE THREAD

**Issue by lleger** [ex-aws/ex_aws_ses#11]
I'm running into a bug with `send_raw_email/2` when trying to set tags, which the AWS docs indicate is legal. I'm wondering if it's a user error on my part or a bug in the library.

```elixir
msg
|> SES.send_raw_email(
  tags:  [%{name: "tag1", value: "tag1value1"}, %{name: "tag2", value: "tag2value1"}]
)
|> ExAws.request()
```

This raises an error:

```elixir
** (ArgumentError) encode_query/1 values cannot be lists, got: [%{name: "tag_1", value: "foo"}, %{name: "tag-2", value: "bar"}]
```

The issue is that the resulting request includes the `Tags` param as well as the encoded `Tag.member.N` params:

```elixir
%ExAws.Operation.Query{
  action: :send_raw_email,
  content_encoding: "identity",
  params: %{
    "Action" => "SendRawEmail",
    "ConfigurationSetName" => nil,
    "RawMessage.Data" => "VG86IGFsaWNlQGV4YW1wbGUuY29tDQpTdWJqZWN0OiA9P3V0Zi04P1E/V2VsY29tZSB0byB0aGUgYXBwLj89DQpSZXBseS1UbzogY2h1Y2tAZXhhbXBsZS5jb20NCk1pbWUtVmVyc2lvbjogMS4wDQpGcm9tOiBib2JAZXhhbXBsZS5jb20NCkNvbnRlbnQtVHlwZTogbXVsdGlwYXJ0L2FsdGVybmF0aXZlOyBib3VuZGFyeT0iOTA4MTk1ODcwOUMwMjlGOTBCRkZGMTMwIg0KQ2M6IGpvaG5AZXhhbXBsZS5jb20NCkJjYzogamFuZUBleGFtcGxlLmNvbQ0KDQotLTkwODE5NTg3MDlDMDI5RjkwQkZGRjEzMA0KQ29udGVudC1UeXBlOiB0ZXh0L3BsYWluDQpDb250ZW50LVRyYW5zZmVyLUVuY29kaW5nOiBxdW90ZWQtcHJpbnRhYmxlDQoNClRoYW5rcyBmb3Igam9pbmluZyENCg0KLS05MDgxOTU4NzA5QzAyOUY5MEJGRkYxMzANCkNvbnRlbnQtVHlwZTogdGV4dC9odG1sDQpDb250ZW50LVRyYW5zZmVyLUVuY29kaW5nOiBxdW90ZWQtcHJpbnRhYmxlDQoNCjxzdHJvbmc+VGhhbmtzIGZvciBqb2luaW5nITwvc3Ryb25nPg0KLS05MDgxOTU4NzA5QzAyOUY5MEJGRkYxMzAtLQ==",
    "Tags" => [%{name: "tag_1", value: "foo"}, %{name: "tag-2", value: "bar"}],
    "Tags.member.1.Name" => "tag_1",
    "Tags.member.1.Value" => "foo",
    "Tags.member.2.Name" => "tag-2",
    "Tags.member.2.Value" => "bar",
    "Template" => nil,
    "TemplateData" => nil
  },
  parser: &ExAws.SES.Parsers.parse/2,
  path: "/",
  service: :ses
}
```

If I look at the implementation for `send_email/4` (which doesn't fail, since the tests pass) I see a call to `build_opts` which drops the offending `Tags` param:

https://github.com/ex-aws/ex_aws_ses/blob/dbe3a3ea76a4570b5f2f11eb414ec29a1a9b60ff/lib/ex_aws/ses.ex#L116

So my thinking is that this also needs to be added to `send_raw_email/2`. When I edit the code and test suite to do this, the tests pass.

Let me know if I'm off base here!

**Comment by lessless** [comment]
Hi @lleger ,

You're absolutely right - `Tags` key shouldn't be there. 
https://docs.aws.amazon.com/ses/latest/APIReference/API_SendRawEmail.html
If you already edited it and tested - please feel free to submit PR. They're hugely welcome 😏 !

**Comment by lleger** [comment]
@lessless Just opened #12 with the code I wrote. My first time digging into this code base, so definitely let me know if anything is off!

**Comment by lessless** [comment]
Closed by #12
